src/components/Menu/MenuItem/__stories__/MenuItem.stories.js

```diff
@@ -6,36 +6,74 @@ import { Activity, Bolt, Settings } from "../../../Icon/Icons";
 import Icon from "../../../Icon/Icon";
 import Tooltip from "../../../Tooltip/Tooltip";
 import DialogContentContainer from "../../../DialogContentContainer/DialogContentContainer";
+import { createStoryMetaSettingsDecorator } from "../../../../storybook/functions/createStoryMetaSettingsDecorator";
 
-export const menuItemTemplate = args => (
+const metaSettings = createStoryMetaSettingsDecorator({
+  component: MenuItem,
+  enumPropNamesArray: ["tooltipPosition"],
+  iconPropNamesArray: ["icon"]
+});
```

Comment:
I'm not sure we need these properties anymore, I think you can provide types in JSDoc and Storybook will use them as a list of values. I think I did it in EditableHeading/Text. Not sure what he `iconPropsNameArray` though

Reply:
`iconPropsNameArray` is giving  you ability to pick icon from dropdown list for specific props
<img width="1051" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/b9149ccc-4e75-469d-823d-91507aed64fb">


Maybe we can think of other way to do it. Anyway I think it's a bit out of scope of this one (this one is already enormous), but feel free to add it to the backlog!